ukrainian/door/factory/factory.md

```diff
@@ -1,3 +1,5 @@
 Ви потрапили до фабрики, що повна скарбів, таємниць та зефіру. Проте, замість
 того щоб дізнатись про таємниці невідомо чого, ви захопили владу на фабриці
 та встановили контроль над усіма запасами зефіру.
+
+[Але нажаль вам не вистачило прихильників утримати фабрику від контролем, вас побили та посадили в кімнату](../../marshmallow.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.